docs/installation/requirements.md

```diff
@@ -87,22 +87,23 @@ Ensure that the time setting in the BIOS of your server is set to the current ti
 
 ## 🖥️ Supported guest OS
 
-Officially supported OS. But there's a lot more than can run on XCP-ng.
+We only officially support operating systems which still receive support from their publisher. But there's a lot more than can run on XCP-ng.
 
 ### Windows based
 
-* Windows Server 2012, 2016, 2019, 2022
+* Windows Server
 * Windows 10
+* Windows 11 (starting with XCP-ng 8.3).
 
 :::info
-Older versions of Windows will run, but in emulation mode (like Windows XP, Server 2003 etc.)
+Very old versions of Windows will run, but only in emulation mode (Windows XP, Windows Server 2003, etc.)
```

Comment:
We should probably tell what emulation means: without any Xen PV tools.

Reply:
Done